.github/e2e-tests-olm/action.yaml

```
@@ -67,6 +67,7 @@ runs:
       shell: bash
       run: |
         # TODO: build a kubernetes-specific bundle to avoid doing this
+        kubectl label nodes -l node-role.kubernetes.io/control-plane=="" node-role.kubernetes.io/infra=""
```

nit: I've tested and this also works, and IMO looks more readable, but feel free to left it as is if you disagree
```suggestion
        kubectl label nodes -l node-role.kubernetes.io/control-plane="" node-role.kubernetes.io/infra=""
```

---

:+1: yup makes sense, this format is also used in the official docs.